Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -170,6 +170,7 @@
 #define WEBPAGE_RELEASE_LOG_ERROR(channel, fmt, ...) RELEASE_LOG_ERROR(channel, "%p - WebPage::" fmt, this, ##__VA_ARGS__)
 
 namespace WebKit {
+using namespace WebCore;
```

Comment:
Why this change?

Reply:
This was to fix the build errors on iOS.